src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/9.diff.sql

```diff
@@ -1666,31 +1666,31 @@ BEGIN
 
         -- ReferencedRequestSequence.Accesionnumber
         INSERT INTO dbo.WorkitemQueryTag (TagKey, TagPath, TagVR)
-        VALUES (NEXT VALUE FOR TagKeySequence, '0040A370.00080050', 'SH')
+        VALUES (NEXT VALUE FOR TagKeySequence, '0040A370.00080050', 'SQ')
 
         -- ReferencedRequestSequence.Requested​Procedure​ID
         INSERT INTO dbo.WorkitemQueryTag (TagKey, TagPath, TagVR)
-        VALUES (NEXT VALUE FOR TagKeySequence, '0040A370.00401001', 'SH')
+        VALUES (NEXT VALUE FOR TagKeySequence, '0040A370.00401001', 'SQ')
```

Comment:
do we want to get rid of `.` to save a bit space, since tagpath always be 8 letter?

Reply:
Yeah we thought about that, but keeping it for now. May be if we support real sequence parsing then we can get rid of all these. We need a major refactoring